ClemBot.Bot/bot/utils/helpers.py

```diff
@@ -1,10 +1,19 @@
-
+from datetime import datetime
 from typing import Generator, Sequence, TypeVar
 
 T = TypeVar("T")
 
+
 def chunk_sequence(sequence: Sequence[T], chunk_size: int) -> Generator[Sequence[T], None, None]:
     """Yield successive chunks from the passed sequence."""
 
     for i in range(0, len(sequence), chunk_size):
-        yield sequence[i : i + chunk_size]
+        yield sequence[i: i + chunk_size]
+
+
+def parse_datetime(time: str) -> datetime:
```

Comment:
Maybe add a docstring saying that this method is for parsing a datetime specifically from the API, same with `format_datetime`.